graphene/src/intersection.rs

```diff
@@ -74,6 +83,7 @@ pub fn intersect_quad_bez_path(quad: Quad, shape: &BezPath, filled: bool) -> boo
 	get_arbitrary_point_on_path(&shape).map(|shape_point| quad.path().contains(shape_point)).unwrap_or_default()
 }
 
+/// Returns any point on `path`.
```

Comment:
I didnt want to get any more specific because there is no **guarantee** that in the future, it will always be the first point on the path, even though it is right now. Would
```
/// Returns a point on `path`.
/// This function will usually return the first point from the path's first segment,
/// but callers should not rely on this behaviour.
``` 
be better?

Reply:
Sounds good :+1: